diff --git a/Code - Insiders/User/settings.json b/Code - Insiders/User/settings.json
index 4518a9f..912cd50 100644
--- a/Code - Insiders/User/settings.json
+++ b/Code - Insiders/User/settings.json
@@ -5,10 +5,22 @@
   "[php]": {
     "editor.defaultFormatter": "kokororin.vscode-phpfmt"
   },
-  "[javascript][javascriptreact][typescript][typescriptreact]": {
+  "[javascript]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
   },
-  "[json][jsonc]": {
+  "[javascriptreact]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[typescript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[typescriptreact]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[json]": {
+    "editor.defaultFormatter": "vscode.json-language-features"
+  },
+  "[jsonc]": {
     "editor.defaultFormatter": "vscode.json-language-features"
   },
   "sync.gist": "376c3dfc48eae5a95d15cfc87639f6cf",
@@ -41,6 +53,7 @@
   "editor.renderWhitespace": "all",
   "editor.renderLineHighlight": "all",
   "editor.detectIndentation": true,
+  "editor.renderIndentGuides": true,
   "editor.guides.indentation": true,
   "editor.guides.bracketPairs": "active",
   "editor.matchBrackets": "always",
@@ -77,11 +90,7 @@
   "editor.minimap.maxColumn": 80,
   "editor.minimap.showSlider": "always",
   "editor.smoothScrolling": true,
-  "editor.quickSuggestions": {
-    "comments": "on",
-    "strings": "on",
-    "other": "on"
-  },
+  "editor.quickSuggestions": true,
   "editor.autoClosingQuotes": "languageDefined",
   "editor.emptySelectionClipboard": false,
   "editor.selectionHighlight": true,
@@ -101,7 +110,7 @@
   "editor.wordBasedSuggestionsMode": "allDocuments",
   "editor.snippetSuggestions": "bottom",
   "editor.inlineSuggest.enabled": true,
-  "editor.inlayHints.enabled": "offUnlessPressed",
+  "editor.inlayHints.enabled": false,
   "editor.hover.enabled": true,
   "editor.hover.above": true,
   "editor.hover.sticky": true,
@@ -178,7 +187,8 @@
   "breadcrumbs.enabled": false,
   "extensions.ignoreRecommendations": true,
   "extensions.autoCheckUpdates": true,
-  "telemetry.telemetryLevel": "off",
+  "telemetry.enableCrashReporter": false,
+  "telemetry.enableTelemetry": false,
   "redhat.telemetry.enabled": false,
   "intelephense.telemetry.enabled": false,
   "code-runner.enableAppInsights": false,
@@ -233,7 +243,7 @@
   "workbench.preferredLightColorTheme": "Quiet Light",
   "workbench.preferredDarkColorTheme": "Monokai",
   "workbench.editorAssociations": {
-    "*.ipynb": "jupyter-notebook"
+    "*.ipynb": "jupyter.notebook.ipynb"
   },
   "workbench.editSessions.autoResume": "onReload",
   "workbench.editSessions.continueOn": "prompt",
@@ -314,7 +324,7 @@
   "typescript.inlayHints.functionLikeReturnTypes.enabled": false,
   "typescript.validate.enable": false,
   "typescript.format.enable": false,
-  "typescript.tsserver.useSyntaxServer": "auto",
+  "typescript.tsserver.useSeparateSyntaxServer": true,
   "typescript.tsserver.experimental.enableProjectDiagnostics": false,
   "typescript.tsserver.maxTsServerMemory": 4096,
   "typescript.tsserver.log": "off",
@@ -656,5 +666,5 @@
   "code-runner.showRunCommandInExplorerContextMenu": false,
   "platformio.enableTelemetry": false,
   "platformio.showHelpInfo": false,
-  "configurationSync.store": {},
+  "configurationSync.store": {}
 }
